### ISSUE TITLE
Keycloak scopes: set to least common or make them configurable

### ISSUE THREAD

**Issue by VJalili** [galaxyproject/galaxy#10166]
The scopes for a keycloak-based authentication are hard-coded and contain `org.cilogon.userinfo`:

https://github.com/galaxyproject/galaxy/blob/2dc39a5c0158387396ec8b2018d29826984780d2/lib/galaxy/authnz/custos_authnz.py#L47

The `org.cilogon.userinfo` scope maybe required for authentication against a CILogon broker; however, it is not a standard keycloak/OIDC scope and it is not included in a fresh installation of a Keycloak server. Additionally, it is not required for a user authentication against IdPs such as Google via Keycloak.

With this scope set, user authentication agains Google via Keycloak fails with the following error: 

```shell
GET /authnz/keycloak/callback?error=invalid_request&error_description=Invalid+scopes%3A+openid+email+profile+org.cilogon.userinfo
```

and Keycloak throws the following error: 

> Invalid parameter value for: scope

I suggest we remove the `org.cilogon.userinfo` scope from `CustosAuthnz` and add option specifying/reading it from configuration.

**Comment by dannon** [comment]
Fixed in #10168

**Comment by VJalili** [comment]
@dannon Thanks for fixing it.
